crates/sui-sdk/examples/event_api.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+// use futures::stream::StreamExt;
+use sui_sdk::rpc_types::EventFilter;
+use sui_sdk::types::digests::TransactionDigest;
+use sui_sdk::SuiClientBuilder;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let sui = SuiClientBuilder::default().build_testnet().await?; // testnet Sui network
+    println!("Sui testnet version{:?}", sui.api_version());
+
+    // TODO - make this work
```

Comment:
is this meant for a subsequent PR?

Reply:
Have a commit that wasn't pushed. Will fix this as it does not work properly at the moment and needs to be revised.